examples/dt/unified/delta/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../../dt/unified/delta/
+
+resources:
+  - ceph_conf_secret.yaml
+  - service-values.yaml
+  - values.yml
```

Comment:
- values.yaml

Not the `yaml` vs `yml`. Please add the 'a' on an update. Otherwise It can't find it.